### PR TITLE
Expose date argument in the CLI

### PR DESCRIPTION
**Pull request by RaphaelMeudec**
This PR gives the opportunity to look for a slot after a given date (for example for people who got the covid in the past 3 months and can't get vaccinated today).

### CODE REVIEW

**Comment by JB-CHAUVIN** [comment]
Great feature, I was doing the same modification myself and saw your code :) 
Thanks you very much ! 
I am trying the code right now and it looks working 👍

**Comment by rbignon** [comment]
Hello,

Thank you for your patch.

--date is ambiguous I think, perhaps something like --from (even if it is not clear too) or --min-date, what do you think?

**Comment by RaphaelMeudec** [comment]
Maybe  `--start-date` seems more appropriate?

**Comment by rbignon** [comment]
Yes `--start-date` is fine.

**Comment by RaphaelMeudec** [comment]
@rbignon ready to go!

File: README.md [hunk 1]
```diff
@@ -103,6 +104,17 @@ Starting to look for vaccine slots for Roger Philibert in 5 next day(s)...
...
 ```
 
+### Look on specific date
+
+By default, the script looks for slots between now and next day at 23:59:59. If you can't be vaccinated right now (e.g covid in the last 3 months or out of town) and you are looking for an appointment in a distant future, you can pass a starting date:
+
+```
+$ ./doctoshotgun.py paris roger.philibert@gmail.com --start-date 17/06/2021
+Password:
+Starting to look for vaccine slots for Roger Philibert in 5 next day(s)...
```
Comment (rbignon) [inline README.md]: By default it is 5 days I guess, and perhaps display in output from which date it starts?

Comment (RaphaelMeudec) [reply README.md]: I modified the log

**Pull request merged**